wagtail/models/__init__.py

```diff
@@ -234,6 +234,10 @@ class RevisionMixin(models.Model):
         editable=False,
     )
 
+    default_exclude_fields_in_copy = [
```

Comment:
Question: Is the intention to allow developers to work with `exclude_fields_in_copy` freely, but guard against getting `latest_revision` in? Should this be made "internal" (i.e. `_default_exclude_fields_in_copy`)?

Finally, it would be great to add a note on how this is used in conjunction with `exclude_fields_in_copy`.

Reply:
> Question: Is the intention to allow developers to work with `exclude_fields_in_copy` freely, but guard against getting `latest_revision` in?

Yep.

> Should this be made "internal" (i.e. `_default_exclude_fields_in_copy`)?

I'd love to, but this is based on how we do it in `Page` and I'm not sure it'd be a good idea to have a different name:

https://github.com/wagtail/wagtail/blob/23f661a48ad2487d8a1107c8dab3b88fea53ed44/wagtail/models/__init__.py#L1187-L1200